isotovideo

```diff
@@ -116,8 +116,16 @@ use commands;
 
 $bmwqemu::vars{BACKEND} ||= "qemu";
 
-# Load the main.pm from the casedir checked by the sanity checks above
-require $bmwqemu::vars{CASEDIR}."/main.pm";
+# Try to load the main.pm from either casedir checked by the sanity checks
+# above or fall back to load from a "distri" subdirectory to allow test
+# distribution collections with multiple distributions or flavors in one
+# repository.
+if (-f "$bmwqemu::vars{CASEDIR}/main.pm") {
+    require $bmwqemu::vars{CASEDIR} . "/main.pm";
+}
+else {
+    require $bmwqemu::vars{CASEDIR} . "/products/$bmwqemu::vars{DISTRI}/main.pm";
+}
```

Comment:
you have to read the discussion in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/920 too ;)